ADAL/src/ui/ADWebAuthController.m

```diff
@@ -488,7 +491,8 @@ + (ADWebAuthController *)sharedInstance
 
 - (BOOL)cancelCurrentWebAuthSessionWithError:(ADAuthenticationError*)error
 {
-    AD_LOG_ERROR(_requestParams.correlationId, @"Application is cancelling current web auth session. error = %@", error);
+    AD_LOG_ERROR(_requestParams.correlationId, @"Application is cancelling current web auth session. error code = %ld", (long)error.code);
+    AD_LOG_ERROR_PII(_requestParams.correlationId, @"Application is cancelling current web auth session. error = %@", error);
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Split this 125 characters long line (which is greater than 120 authorized). [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=objc%3ALineLength)